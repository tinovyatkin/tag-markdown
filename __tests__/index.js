'use strict';

const md = require('../')();

describe('Markdown tag', () => {
  test('single line should have no wrapper', () => {
    expect(
      md`
This is test _markdown_ string :tada:
      `,
    ).toMatchSnapshot();
  });

  test('multiple lines must be wrapped in <p>', () => {
    expect(md`
      # This is header

      This is firs (C) _paragraph_

      This is second :phone: paragraph
    `).toMatchSnapshot();
  });
});
