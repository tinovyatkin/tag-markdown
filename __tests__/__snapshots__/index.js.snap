// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Markdown tag multiple lines must be wrapped in <p> 1`] = `
"<h1>This is header</h1>
<p>This is firs © <em>paragraph</em></p>
<p>This is second ☎️ paragraph</p>
"
`;

exports[`Markdown tag single line should have no wrapper 1`] = `"This is test <em>markdown</em> string 🎉"`;
